Examples/Scripts/Python/itk_seeding.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+import os
+import argparse
+
+import acts
+import acts.examples
+
+from acts.examples import CsvSpacePointReader
+
+u = acts.UnitConstants
+
+
+def runITkSeeding(field, outputDir, s=None):
+
+    # Read input space points from input csv files
+    evReader = CsvSpacePointReader(
+        level=acts.logging.INFO,
+        inputStem="spacepoints",
+        inputCollection="pixel",
+        inputDir="acts/Examples/Scripts/Python/data_itk_seeding/CsvSpacePointsOutput_singleMu_100GeV_300evnts/",
+        outputSpacePoints="PixelSpacePoints",
+    )
+
+    gridConfig = acts.SpacePointGridConfig(
+        bFieldInZ=1.997244311 * u.T,
+        minPt=900 * u.MeV,
+        rMax=320 * u.mm,  # pixel: 320 mm, strip: 1000 mm
+        zMax=3000 * u.mm,
+        zMin=-3000 * u.mm,
+        deltaRMax=280 * u.mm,  # pixel: 280 mm, strip: 600 mm
+        cotThetaMax=27.2899,  # pixel: 27.2899 (4 eta), strip: 900
```

Comment:
what does strip:900 mean?

Reply:
In Athena the eta cut is not applied for the strip SPs, so the strip:900 is also just a large value. I will remove this part of the comment and when I start checking the strip and overlap SPs I can set a better value for this